parsers/to-css-text-style/to-css-text-style.spec.ts

```diff
@@ -17,7 +17,7 @@ describe('to-css-text-style', () => {
         '.ts-with-allan-again {\n' +
```

Comment:
I don't understand why this text style is named like this. Please rename it if the user is supposed to see and read this file. E.g; ".text-style-1"

Reply:
Seeds are not the subject